submissions/olabodeAmusan/Database/init2.sql

```diff
@@ -0,0 +1,34 @@
+CREATE DATABASE transportation_company;
+USE transportation_company;
+CREATE TABLE passengers(
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(250) NOT NULL,
+sex ENUM('male','female') NOT NULL DEFAULT 'male',
+age INT NOT NULL DEFAULT '0',
```

Comment:
Change line 7 to : NULL in place of NOT NULL
Because some passengers, however, do not like sharing their age information.